client/app/core/core.module.js

```diff
@@ -22,7 +22,7 @@ import { DialogFieldRefreshFactory } from './dialog-field-refresh.service.js'
 import { EventNotificationsFactory } from './event-notifications.service.js'
 import { ExceptionModule } from './exception/exception.module.js'
 import { LanguageFactory } from './language.service.js'
-import { LanguageSwitcherDirective } from './language-switcher/language-switcher.directive.js'
+import { LanguageSwitcherComponent } from './language-switcher/language-switcher.component.js'
```

Comment:
Thoughts on moving this to `./app/components`?

Reply:
I thought we were going to get rid of that folder, how about /app/shared ?

Reply:
OOoooo Shared kinda makes sense... it is after all used in two distinct places, login.html and application.html (one is a state the latter a layout)

Reply:
moved the component to /app/shared